recipes/libpng/all/conandata.yml

```
@@ -2,7 +2,19 @@ sources:
   "1.6.37":
     url: "https://github.com/glennrp/libpng/archive/v1.6.37.tar.gz"
     sha256: "ca74a0dace179a8422187671aee97dd3892b53e168627145271cad5b5ac81307"
+  "1.5.2":
+    url: "https://github.com/glennrp/libpng/archive/v1.5.2.tar.gz"
+    sha256: "6b27e93ff12993fc76c6c7bee4f4be49eac71984af55f0dee45ab2b70fa090e7"
 patches:
   "1.6.37":
     - patch_file: "patches/CMakeLists-zlib.patch"
       base_path: "source_subfolder"
+  "1.5.2":
+    - patch_file: "patches/CMakeLists-symlink.patch"
+      patch_description: "Fix symlink macro call for Linux debug builds. Previously it was attempting to symlink libpng to libpng15 that did not exist in debug builds."
+      patch_type: "backport"
```

It is more of a portability now 👍 

```suggestion
      patch_type: "portability"
```